src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/ApprovalsOuterApiClient.cs

```diff
@@ -22,14 +21,17 @@ public ApprovalsOuterApiClient(IApprovalsOuterApiHttpClient client)
                 ($"providers/{providerId}/courses?trainingCode={courseCode}&accountLegalEntityId={accountLegalEntityId}",
                 cancellationToken: cancellationToken);
         }
-        public Task<GetHasPermissionResponse> GetHasPermission(long? ukPrn, long? accountLegalEntityId, Operation operation)
+
+        public async Task<GetHasPermissionResponse> GetHasPermission(long ukprn)
```

Comment:
It's worth renaming everything named GetHasPermission to GetHasRelationshipWithPermission. Something I missed when I missed implementing these methods, but the endpoint in outer is no longer Has, so this name is slightly misleading

Reply:
Fixed